core/src/verified_vote_packets.rs

```diff
@@ -3,20 +3,13 @@ use crate::{
     result::Result,
 };
 use solana_perf::packet::Packets;
-use std::{collections::HashMap, ops::Deref, time::Duration};
+use std::{collections::HashMap, time::Duration};
 
 #[derive(Default)]
 pub struct VerifiedVotePackets(HashMap<CrdsValueLabel, (u64, Packets)>);
 
-impl Deref for VerifiedVotePackets {
-    type Target = HashMap<CrdsValueLabel, (u64, Packets)>;
-    fn deref(&self) -> &Self::Target {
-        &self.0
-    }
-}
-
 impl VerifiedVotePackets {
-    pub fn get_and_process_vote_packets(
+    pub fn receive_and_process_vote_packets(
```

Comment:
nits: `get_` => `receive_`.

Reply:
Also, I wanted to define real getter called `get_vote_packets` and wanted to avoid the same `get_` for vastly different tasks. :)